com.protonvpn.www.yml

```diff
@@ -0,0 +1,513 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+app-id: com.protonvpn.www
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: protonvpn
+separate-locales: false
```

Comment:
Is there any rationale for this? If not, its better to not add this line.

Reply:
An empty extension does no harm. If a year from now somebody adds a library that happens to have translations it should be a separate locale.